### PR TITLE
docs: note '--data-urlencode' option

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by bagder** [comment]
I suppose we call it "URL encode" in plain English?

File: .github/scripts/spellcheck.words [hunk 1]
```diff
@@ -810,6 +810,7 @@ URI
 URIs
 url
 URL's
+urlencode
```
Comment (bagder) [inline .github/scripts/spellcheck.words:813]: I disagree. This is not an English word. You would write it as URL encode.

Comment (andy5995) [reply .github/scripts/spellcheck.words:813]: Ok, I'll make the change. But 'urlencoded' is used elsewhere in the document and already on the whitelist.

Comment (bagder) [reply .github/scripts/spellcheck.words]: Yeah, I noticed. But I think maybe those should rather be fixed as well at some point?

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!
